lib/router.js

```diff
@@ -19,7 +19,7 @@ Router.prototype.methods = ['get', 'post', 'put', 'delete', 'head',
 Router.prototype.add = function r_add(method, path, handler) {
   method = method.toLowerCase();
   if (this.methods.indexOf(method) === -1) {
-    throw new Error('Method %s is not supported', method);
```

Comment:
Ok, change by `'Method', method, 'is not supported'` instead of getting rid of. Anyway, asking @arcturus as this is a matter of implementation.